### ISSUE TITLE
Can this be used in place of the 1.2.0 version?

### ISSUE THREAD

**Issue by mdinescu** [networking/sonata-admin-annotation-bundle#1]
Looks like this version is based off of the 2.x version of `sonata-admin-annotation-bundle` (maybe the 2.4 version?)

I'm looking for a replacement for the 1.2.0 version that was used in the 2.3 init-cms

Do you know if this is compatible with 1.2.0?

Essentially, I'm thinking of adding something like this in composer.json to force it to use this version:

```
	"ibrows/sonata-admin-annotation-bundle": "dev-master as 1.2.0"
    },
    "repositories": [
        {
            "type": "vcs",
            "url": "git@github.com:networking/annotation-reader.git"
        },
        {
            "type": "vcs",
            "url": "git@github.com:networking/sonata-admin-annotation-bundle.git"
        },
        {
            "type": "vcs",
            "url": "git@github.com:networking/sonata-translation-bundle.git"
        }
    ],
```

**Comment by ychadwick** [comment]
Hi there, the bundle we use is the only one. The original agency, ibrows, was bought by rice Waterhouse Coopers and the deleted all the github repositories. The  version We have was the last version of it, and we have now completely integrated it into our cm/s in our version 4.4.

If you use the old initcms 2.3 bundle, you can use the the version from us, which is renamed copy for legacy projects. You will need to write a lot of aliases in your composer file.

**Comment by mdinescu** [comment]
yes, it's really painful.. it's a really old app though that hasn't had any serious updated for ~6 years, but at the time when it was developed it had some customizations so going from 2.3 to a newer version would also probably require a lot of work
